src/auxinfo/auxinfo_commit.rs

```diff
@@ -123,6 +125,9 @@ impl CommitmentScheme {
         // Public parameters in this decommit must be consistent with each other
         scheme.clone().public_key.verify(context)?;
 
+        let lower_bound = BigNumber::one() << ELL;
+        assert!(scheme.public_key.params().scheme().modulus() >= &lower_bound);
```

Comment:
issue: `assert!` will panic if it fails; we have to handle all errors in production code. 

Also, I think we want to check that the modulus is an exact number of bits. Too small is a known problem for proof reasons, as we've discussed, but I also don't really want to get dos'd by getting giant moduli from someone.